libs/full/agas/CMakeLists.txt

```
@@ -0,0 +1,40 @@
+# Copyright (c) 2019-2020 The STE||AR-Group
+#
+# SPDX-License-Identifier: BSL-1.0
+# Distributed under the Boost Software License, Version 1.0. (See accompanying
+# file LICENSE_1_0.txt or copy at http://www.boost.org/LICENSE_1_0.txt)
+
+cmake_minimum_required(VERSION 3.13 FATAL_ERROR)
+
```

@hkaiser I think this is all that you're missing:
```suggestion
if(NOT HPX_WITH_DISTRIBUTED_RUNTIME)
  return()
endif()
```
All the failing builds have the distributed runtime off: https://github.com/STEllAR-GROUP/hpx/blob/ef855bcdff1e0889e1cfd680c5cd30aefc273611/.jenkins/cscs/env-gcc-newest.sh#L32 and https://github.com/STEllAR-GROUP/hpx/blob/ef855bcdff1e0889e1cfd680c5cd30aefc273611/.jenkins/lsu/env-gcc-9-cuda-11.sh#L21.

---

Ahh, make sense. I was somehow assuming that all modules in `full` are automatically disabled for the local runtime. I'll apply the change. Thanks!